universal-application-tool-0.0.1/app/views/admin/AdminLayout.java

```diff
@@ -47,7 +49,11 @@ protected Content render(HtmlBundle bundle, boolean isCentered) {
 
   @Override
   public HtmlBundle getBundle(HtmlBundle bundle) {
-    return super.getBundle(bundle).addHeaderContent(renderNavBar());
+    bundle = super.getBundle(bundle).addHeaderContent(renderNavBar());
```

Comment:
Since order is unimportant I'd leave `getBundle` unchanged and add the scripts in `render`.

Reply:
moved